tools/SetupFlow/DevHome.SetupFlow.Common/DevDriveFormatter/DevDriveFormatter.cs

```diff
@@ -16,6 +16,11 @@ namespace DevHome.SetupFlow.Common.DevDriveFormatter;
 /// </summary>
 public class DevDriveFormatter
 {
+    /// <summary>
+    /// Default allocation unit size for Dev Drives created via a virtual hard disk.
+    /// </summary>
+    private const uint FourKb = 4096;

```

Comment:
Since this is dealing with drive space, would it be better to explicitly declare the size of the uint?

Reply:
when you say "explicitly declare the size of the uint" what do you mean by explicitly?

Reply:
`Uint16`, `Uint32`, or `Uint64`  However, on second thought, maybe not since uint will resolve to the correct size.

I'm just wondering if this'll ever get into an issue where the two sizes don't match.  That'll be rare though.  NVM, don't bother with it.